tests/suites/test_suite_psa_crypto.data

```diff
@@ -6257,6 +6257,87 @@ PSA key derivation: over capacity 42: output 43+1
 depends_on:PSA_WANT_ALG_HKDF:PSA_WANT_ALG_SHA_256
 derive_output:PSA_ALG_HKDF(PSA_ALG_SHA_256):PSA_KEY_DERIVATION_INPUT_SALT:"000102030405060708090a0b0c":PSA_SUCCESS:PSA_KEY_DERIVATION_INPUT_SECRET:"0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b":PSA_SUCCESS:PSA_KEY_DERIVATION_INPUT_INFO:"f0f1f2f3f4f5f6f7f8f9":PSA_SUCCESS:0:"":PSA_SUCCESS:"":42:"3cb25f25faacd57a90434f64d0362f2a2d2d0a90cf1a5a4c5db02d56ecc4c5bf34007208d5b887185865ff":"ff":0:1:0
 
```

Comment:
I just realized we're missing a bad test case: PBKDF2 with all valid parameters except that cost=0. Please add one.

Reply:
Never mind that, https://github.com/Mbed-TLS/mbedtls/pull/7702 would be a better place for it.